guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md

```diff
@@ -143,18 +159,19 @@ Example:
 
 #### `constraint` subnode
 
-The `constraint` subnode is represented by 3 different types:
+The `constraint` subnode always contains the following parameters:
 
-* `primary`
-* `unique`
-* `foreign`
+Attribute | Description
+--- | ---
+`type` | One of `primary`, `unique`, or `foreign`
+`referenceId` |  In most instances, a system-generated reference ID that represents an entity in the schema. System-generated IDs cannot be changed. However, you can override `referenceId` values.
```

Comment:
This is uncorrect. We have to change it to something like that:
This is a custom identifier that is used only for relation mapping in the scope of Declarative Schema declarations - db_schema.xml files. The real entity in the database will have a system-generated name. The most convenient way - to set the value of this attribute the same as the system-generated name. You can get thesenames from db_schema_whitelist.json as the result of execution `generate-whitelist` command [https://devdocs.magedevteam.com/kh_schema-updates/guides/v2.3/extension-dev-guide/declarative-schema/migration-commands.html#create-a-schema-whitelist]